benchmarks/IO.py

```diff
@@ -11,14 +11,26 @@
     "int64",
     "float64",
     "uint64",
+    "str"
 )
+COMPRESSIONS = (
+    "none",
+    "snappy",
+    "gzip",
+    "brotli",
+    "zstd",
+    "lz4"
+)
+
 
+def time_ak_write(N_per_locale, numfiles, trials, dtype, path, seed, parquet, comps=None):
+    if comps is None or comps == [""]:
+        comps = COMPRESSIONS
 
-def time_ak_write(N_per_locale, numfiles, trials, dtype, path, seed, parquet, compressed=False):
     if not parquet:
-        print(">>> arkouda {} HDF5 write with compressed={}".format(dtype, compressed))
+        print(">>> arkouda {} HDF5 write with compression={}".format(dtype, comps))
```

Comment:
Do we want to be printing out the compressions with HDF5 since it isn't supported?